packages/utils/selectors/k8s.ts

```diff
@@ -1,12 +1,16 @@
 import { K8sResourceCommon } from "@openshift-console/dynamic-plugin-sdk";
 import * as _ from 'lodash';
-import { GetAPIVersionForModel } from "../../types";
 
-type GetStringProperty<T = K8sResourceCommon> = (resource: T) => string;
+type GetStringProperty<T = K8sResourceCommon & { provider?: string, buckets?: number }> = (resource: T) => string;
```

Comment:
```suggestion
type GetStringProperty<T = K8sResourceCommon> = (resource: T) => string;
```

Reply:
these I have added in order to display it in the table